lib/4_filter_numbers.rb

```diff
@@ -5,5 +5,12 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def filter_numbers(arr)
-  # Write your code here.
+  arr.each { |x|
+  if x >= 10
+      arr.delete(x)
+  end}
+  arr.each { |x|
```

Comment:
I appreciate the thought behind this addition but I think it's unnecessary. Even if you have an Array with a mixture of Integers and Floats, lets say `[1, 2, 3.5, 9, 10.01, 11, 15.4]` those numbers can still all be compared successfully against the Integer `10`.